src/test/java/LoginTests.java

```diff
@@ -1,101 +1,38 @@
 import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.chrome.ChromeOptions;
 import org.testng.Assert;
 import org.testng.annotations.Test;
-
-import java.time.Duration;
+import pages.LoginPage;
 
 public class LoginTests extends BaseTest {
-    @Test
-    public void successfulLoginTest()  {
-
-//      Added ChromeOptions argument below to fix websocket error
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
-
-        WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+    LoginPage loginPage = new LoginPage();
+    // locators
+    By logOutBtn = By.cssSelector(".fa.fa-sign-out");
+    By submitLoginBtn = By.cssSelector("button[type='submit']");
+    By registrationLink = By.cssSelector("a[type = 'submit']");
+    By registrationBtn = By.id("button");
```

Comment:
Locators (by POM definition) should be stored in the page objects themselves. 

![Screenshot_1](https://user-images.githubusercontent.com/104761482/230733546-75362ffd-acc4-4888-8ea1-b47f7a223ba8.jpg)